### ISSUE TITLE
Bug: Schematron validation IOP checks not separated

### ISSUE THREAD

**Issue by Romain-Bernier** [Dash-Industry-Forum/DASH-IF-Conformance#496]
Hello, 
The manifest in attachment failed during the Schematron validation and I don't understand the error message. Can you explain the issue ? 
[mpd_example.txt](https://github.com/Dash-Industry-Forum/DASH-IF-Conformance/files/4293936/mpd_example.txt)
thanks

**Comment by eceozturk** [comment]
In the AdaptationSet with @id attribute set to 23, you have a ContentProtection element:

```
<ContentProtection schemeIdUri="urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed">
         <cenc:pssh>***</cenc:pssh>
</ContentProtection>>
```

There is one ">" extra character. When removed, it should be fine.

**Comment by Romain-Bernier** [comment]
The extra ">" came from a copy mistake, sorry for that.
I retry without it, I have the same unclear error : 

location="/*:MPD[namespace-uri()='urn:mpeg:dash:schema:mpd:2011'][1]/*:Period[namespace uri()='urn:mpeg:dash:schema:mpd:2011'][1]/*:AdaptationSet[namespace uri()='urn:mpeg:dash:schema:mpd:2011'][1]/*:Representation[namespace-uri()='urn:mpeg:dash:schema:mpd:2011'][1]/*:ContentProtection[namespace-uri()='urn:mpeg:dash:schema:mpd:2011'][1]">

The ContentProtection descriptors shall always be present in the AdaptationSet element and apply to all contained Representations.

[mpd_example2.txt](https://github.com/Dash-Industry-Forum/DASH-IF-Conformance/files/4297226/mpd_example2.txt)

**Comment by eceozturk** [comment]
It is not the same error as it passes the MPD Validation now and getting stuck at the Schematron Validation. 

It seems that this check should be done in case of validation against the IOP profile. However the schematron check lacks this condition. So, this is a bug and I change the issue title to reflect this.

In the meantime (until this is fixed), it seems that you are applying the same ContentProtection to each Representation; so if you would like to go on with the Segment Validation part of the conformance (where the DASH media content in each Representation will be downloaded and validated), you can simply remove all occurances of the ContentProtection element that are on the Representation level and put them on the Adaptation Set level. So, your MPD would be: 

[mpd_example3.txt](https://github.com/Dash-Industry-Forum/DASH-IF-Conformance/files/4297351/mpd_example3.txt)

**Comment by Romain-Bernier** [comment]
Ok, thanks for your reply and the upcoming bugfix.
I put the ContentProtection element inside the representation because some time, we have differents keys for differents representations.

**Comment by eceozturk** [comment]
This is fixed in 7189cad36964c2536fd9d97bf10796bd610d165d
Closing the issue now. Please feel free to reopen/contact if further issues.

**Comment by Romain-Bernier** [comment]
Thanks, it's working well
